app/services/notifications/new_comment/send.rb

```diff
@@ -36,7 +36,7 @@ def call
 
         # Send PNs using Rpush - respecting users' notificaton delivery settings
         targets = User.joins(:notification_setting)
-          .where(notification_setting: { mobile_comment_notifications: true }).ids
+          .where(id: user_ids, notification_setting: { mobile_comment_notifications: true }).ids
```

Comment:
the filter query by user_ids (in the set, created earlier) behavior was present in the [initial version](
https://github.com/forem/forem/commit/e87dead7ade397975208d1bbf853db5d0d5d336b#diff-56d67da7087534bc7d4cffd953e00dc2e3eb41047f1e82f3124cd6d69c509873L38) and this restores that.